runtime/service_test.go

```diff
@@ -74,12 +74,13 @@ func TestBuildVMConfiguration(t *testing.T) {
 			expectedStubDriveCount: 1,
 		},
 		{
-			name: "Input",
+			name: "Only Request",
```

Comment:
Can we have a testcase where the `IsReadOnly` flag is unset?

Reply:
Like the above comment, `IsReadOnly` is either true or false.

Reply:
https://github.com/firecracker-microvm/firecracker-containerd/pull/283/commits/6690f80e0383c6061290e93ac461489860565a20 adds a new test case that doesn't specify IsWritable.